### PR TITLE
Remove alternatives

### PR DESCRIPTION
**Pull request by porunov**
@sroegner , @vutny ,
It is the PR which was discussed in #38 PR.
This PR removes alternatives to prevent bugs and painful maintenance.

### CODE REVIEW

File: zookeeper/uninstalled.sls [hunk 1]
```diff
@@ -28,13 +28,3 @@ zookeeper-reload-systemctl:
     - onchanges:
       - file: {{ zk.systemd_unit }}
 {%- endif %}
```
Comment (sroegner) [inline zookeeper/uninstalled.sls]: Looks to me like the symlinks remain in place after uninstall?

Comment (porunov) [inline zookeeper/uninstalled.sls]: No, it will be removed because of `zk-directories-removed` state.

Comment (porunov) [inline zookeeper/uninstalled.sls]: This thing just checks when the file changes. For example, when the file content changes or the file is deleted, then the state will be executed. But the state will be executed only once.

**Comment by javierbertoli** [comment]
@vutny, @porunov, @sroegner, do you want me to merge?

**Comment by vutny** [comment]
@javierbertoli Why not :-)

**Comment by javierbertoli** [comment]
Done! 👍

**Comment by sroegner** [comment]
Thanks @javierbertoli

**Pull request merged**